Gruntfile.coffee

```diff
@@ -0,0 +1,339 @@
+          dest + src.substr(startOfImagesDir + uniqueFolderToSearchFor.length);
+      js:
+        expand: true
+        updateAndDelete: true
+        dest: '<%= webAssetsPath %>/js/'
+        filter: 'isFile'
+        src:  [
+          'src/**/Resources/assets/js/**'
+        ]
+        rename: (dest, src) ->
+          uniqueFolderToSearchFor = '/js/'
+          startOfImagesDir = src.indexOf(uniqueFolderToSearchFor)
+          dest + src.substr(startOfImagesDir + uniqueFolderToSearchFor.length);
+      sass:
+        expand: true
+        updateAndDelete: true
+        dest: '<%= assetsPath %>/sass/'
+        filter: 'isFile'
+        src:  [
+          'src/**/Resources/assets/sass/**'
+        ]
+        rename: (dest, src) ->
+          uniqueFolderToSearchFor = '/sass/'
+          startOfImagesDir = src.indexOf(uniqueFolderToSearchFor)
+          dest + src.substr(startOfImagesDir + uniqueFolderToSearchFor.length);
+
+    # Watch configuration
+    watch:
+      # Watch the coffee files so we can (re)generate the js files
+      coffee:
+        files: [
+          '<%= assetsPath %>/coffee/**'
+          'src/**/Resources/assets/coffee/**'
+        ]
+        tasks: [
+          'sync:coffee'
+          'generateJs'
+        ]
+        options:
+          livereload: true
+      # Watch the files to be copied
+      copy:
+        files: [
+          'src/**/Resources/assets/images/**'
+          'src/**/Resources/assets/js/**'
+        ]
+        tasks: [
+          'concurrent:syncImagesAndJs'
+        ]
+        options:
+          livereload: true
+      # Watch the font folders so we can (re)generate the fonts
+      fonts:
+        files: [
+          'src/**/Resources/assets/fonts/**'
+        ]
+        tasks: [
+          'generateFonts'
+        ]
+        options:
+          livereload: true
+      # Watch the icon files so we can (re)generate the iconfont
+      icons:
+        files: [
+          'src/**/Resources/assets/icons/**'
+        ]
+        tasks: [
+          'generateIconFonts'
+        ]
+        options:
+          livereload: true
+      # Watch the sass files so we can (re)generate the css files
+      sass:
+        files: [
+          '<%= assetsPath %>/sass/**'
+          'src/**/Resources/assets/sass/**'
+        ]
+        tasks: [
+          'sync:sass'
+          'compass:dev'
+          'autoprefixer'
+        ]
+        options:
+          livereload: true
+
+  # Load all grunt tasks
+  require('load-grunt-tasks')(grunt);
+
+  # Generate all needed files for the fonts and do some cleanup
+  grunt.registerTask 'generateFonts', [
+    'fontgen'
+    'copy:glyphiconsWoff2'
+    'clean:afterFontgen'
+  ]
+
+  grunt.registerTask 'generateIconFonts', [
+    'clean:beforeGenerateIconFonts'
+    'webfont'
+  ]
+
+  # Generate the css-files
+  grunt.registerTask 'generateCssForDev', [
+    'compass'
+    'autoprefixer'
+    'clean:afterGenerateCssForProduction'
```

Comment:
Je voert die zowel voor dev als voor productie uit. Is dit dan niet gewoon "afterGenerateCss"?